cpu/ppc64_cpu_test.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+
+"""
+Test to verify ppc64_cpu command.
+"""
+
+from avocado import Test
+from avocado import main
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class PPC64Test(Test):
+    """
+    Test to verify ppc64_cpu command for different supported values.
+    """
+
+    def setUp(self):
+        """
+        Verifies if powerpc-utils is installed, and gets current SMT value.
+        """
+        command = "uname -p"
+        if 'ppc' not in process.system_output(command, ignore_status=True):
+            self.skip("Processor is not ppc64")
+        if SoftwareManager().check_installed("powerpc-utils") is False:
+            if SoftwareManager().install("powerpc-utils") is False:
+                self.skip("powerpc-utils is not installing")
```

Comment:
Just a thought why we can't package check this way 
        if not (SoftwareManager().check_installed("powerpc-utils")
                or  SoftwareManager().install("powerpc-utils")) :
a suggestion